packages/lib/src/settings/options/allowSlideshowCounter.js

```diff
@@ -3,6 +3,8 @@ import { INPUT_TYPES } from '../utils/constants';
 export default {
   title: 'Show Slideshow Counter',
   isRelevant: (sp) => sp.oneRow && sp.isAutoSlideshow,
+  isRelevantDescription:
+    'Set a Horizontal scrolled gallery, set "Auto Slide" to "true".',
```

Comment:
write the exact string to search (Scroll Direction)

Reply:
Set a/an Horizontal gallery ("Scroll Direction" as "Horizontal") and set "Auto Slide" to "true"